index.html

```diff
@@ -481,6 +464,29 @@ <h2>
           going to change at all.
         </p>
       </section>
+      <section>
+        <h2>
+          Common Safety Checks
+        </h2>
+        <p>
+          The <dfn>common safety checks</dfn> for a {{Document}}
+          |document:Document| are the following steps:
+        </p>
+        <ol class="algorithm">
+          <li data-tests="non-fully-active.html">If |document| is not
+          [=Document/fully active=], [=exception/throw=] an
+          {{"InvalidStateError"}} {{DOMException}}.
+          </li>
+          <li data-tests="lock-sandboxed-iframe.html">If |document| has the
+          [=sandboxed orientation lock browsing context flag=] set,
+          [=exception/throw=] {{"SecurityError"}} {{DOMException}}.
+          </li>
+          <li data-tests="hidden_document.html">If |document|'s
+          [=Document/visibility state=] is "hidden", [=exception/throw=]
+          {{"SecurityError"}} {{DOMException}}.
+          </li>
```

Comment:
It strikes me that if you adopt https://github.com/whatwg/html/commit/2db564c6902ace8c2539560e508ce22c2948c4d8 here (and I think you should in a follow-up) you won't get to distinguish exceptions, but that's probably okay.

Reply:
Yeah, that's ok. And yeah, will do that as a followup.